docs/PokemonCreation.md

```diff
@@ -0,0 +1,60 @@
+## Pokemon Creation
+
+The two main files that handle Pokemon creation are [`Equations`](../src/components/Equations.ts) and [`Utilities`](../src/components/Utilities.ts), both of which incorporate ['numberMaker'](../node_modules/numbermakr/lib/NumberMakr.d.ts) for random generation of numbers.
+`Equations` includes functions for choosing random wild Pokemon and for creating the actual Pokemon object. `Utilities` facilitates the creating of a wild Pokemon by preparing parameters then calling `Equations` creation of the Pokemon object.
+
+A newly created Pokemon's member types can be found in [`Battles`](../src/components/Battles.ts) under the IPokemon interface.
+
+### Unit Test and Trainer Battles
+
+The steps for creating a Pokemon in a unit test and trainer battle are:
+1. Pass into the newPokemon function from `Equations` an object with appropriate members:
+
+```javascript
+const pokemonTitle: string[] = "CHARMANDER".split("");
+const pokemonLevel: number = 10;
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    level: undefined || pokemonLevel,
+    title: pokemonTitle
+});
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    item: "Potion".split(""),
+    level: undefined || pokemonLevel,
+    moves: [{
+        title: "Ember",
+        remaining: FSP.constants.moves.Ember.pp,
+        uses: FSP.constants.moves.Ember.pp}],
+    title: pokemonTitle
+});
+```
+
+### Wild Pokemon Grass Encounter
+
+The steps for creating a Pokemon in a wild Pokemon grass encounter are:
```

Comment:
For 1, how does it use the grass IMap and IArea? 
For 3 and 4, what is `createPokemon`? 

> is called returning the wild encountered Pokemon

Confusing. After reading this page, it's not clear what the differences between newPokemon and createPokemon are.